test/evergreen/dump_wt_datafiles.sh

```diff
@@ -0,0 +1,100 @@
+#! /usr/bin/env bash
+#
+# This script is used to list and dump WiredTiger data files ('WiredTiger.wt') generated by test/format program.
+#
+# Supported return values:
+#   0: list & dump are successful for all data files
+#   1: list is not successful for some data file
+#   2: dump is not successful for some table object
+#   3: 't' or 'wt' binary file does not exist
+#   4: 'WT_TEST' directories do not exist
+#   5: Command argument setting is wrong
+#   6: No table is dumped out from all data files
+
+set -eu
+
+verbose=false
+
+if [ $# -gt 1 ]; then
+	echo "Usage: $0 [-v]"
+	exit 5
+elif [ $# -eq 1 ]; then
+        [ "$1" == "-v" ] && verbose=true || verbose=false
+fi
+
+# Switch to the Git repo toplevel directory
+cd $(git rev-parse --show-toplevel)
+
+# Walk into the test/format directory in which data files are generated
+cd build_posix/test/format
+
+# Check the existence of 'WT_TEST' directories
+num_dirs=$(find . -type d -name 'WT_TEST.[0-9]*' | wc -l)
+if [ "${num_dirs}" -eq "0" ]; then
+	echo "test/format 'WT_TEST' directories do not exist, exiting ..."
+	exit 4
+fi
+
+# Check the existence of 'wt' binary
+wt_binary=$(find ../../ -type f -executable -name wt | grep '.libs' | head -1)
+if [ ! -x "${wt_binary}" ]; then
```

Comment:
I wouldn't do a `find`, we know where the `wt` binary is built,  and if someone creates a file named `wt` in the future, this will break.

Reply:
Fair enough, will replace `find` command with the path of the `wt` binary.